pipelines/cpSky.yaml

```diff
@@ -0,0 +1,18 @@
+description: Sky frame generation for ci_cpp_gen3
+instrument: lsst.obs.lsst.Latiss
```

Comment:
Does this mean that this pipeline is specific for LATISS alone, and should not be used with other datasets?

Reply:
This should almost certainly get migrated to `ci_cpp`, and loaded from the Latiss pipeline subdirectory.